packages/frontend/src/page/document_page.css

```diff
@@ -1,3 +1,20 @@
+.warning-banner {
```

Comment:
Just to note that we have generic "alert" components in `src/component/alert.tsx`. I'm not attached to the styling currently there (which IIRC I borrowed from GitHub), but we should consider consolidating this stuff to ensure a consistent look and feel. Happy to defer that to a future PR, though.